src/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,37 @@
+.search-bar {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  padding: 20px;
+  background-color: #EFEFEF;
+}
+
+.search-bar__input-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.search-bar__input-container--zip {
+  width: 22%;
```

Comment:
A quick note for styling, we should decide as a team which measurements we should use for font-size, widths, etc.  I think it would be good to keep things consistent between the three of us.